javascript/statements.json

```
@@ -2128,6 +2128,45 @@
           }
         }
       },
+      "using_keyword": {
```

Why not just `using`?

```suggestion
      "using": {
```

---

Yes, sorry, this was generated from an outdated local collector version.